opencti-platform/opencti-front/src/components/dataGrid/dataTableUtils.tsx

```diff
@@ -494,14 +496,49 @@ const defaultColumns: DataTableProps['dataColumns'] = {
     label: 'Value',
     percentWidth: 20,
     isSortable: false,
-    render: ({ observable_value }, helpers) => defaultRender(observable_value, helpers),
+    // Please check the String.jsx->renderObservableValue. It should have the same behavior and will replace it at the end.
+    render: (observable, helpers) => {
+      switch (observable.entity_type) {
+        case 'IPv4-Addr':
+        case 'IPv6-Addr': {
+          const country = observable.countries?.edges?.at(0)?.node;
+          if (country) {
+            const flag = R.head(
+              (country.x_opencti_aliases ?? []).filter((n: string) => n.length === 2),
+            );
```

Comment:
```suggestion
            const flag = (country.x_opencti_aliases ?? []).filter((n: string) => n.length === 2)[0];
```
Ramda head is not exactly rocket science :)

Reply:
Done